src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow.java

```diff
@@ -199,10 +194,13 @@ public boolean isInTheSameState(double wantedAng) {
     }
 
     public static double getStaticFeedForward(double extenderLength,double elbowAngle) {
-        return (RobotMap.TelescopicArm.Elbow.MIN_Kg + (((RobotMap.TelescopicArm.Elbow.MAX_Kg - RobotMap.TelescopicArm.Elbow.MIN_Kg) * extenderLength)
+        return (RobotMap.TelescopicArm.Elbow.MIN_Kg + (((RobotMap.TelescopicArm.Elbow.MAX_Kg - RobotMap.TelescopicArm.Elbow.MIN_Kg + kS) * extenderLength)
```

Comment:
ks needs to be applied in the direction of the movement and should be 0 if the goal is to be stationary which is the goal of this function

Reply:
i think that you need to remove the Ks because Kg takes care of that

Reply:
deleted ks